src/tests/jobtest.c

```diff
@@ -18,17 +18,26 @@
 #include <daos_mgmt.h>
 #include <daos/common.h>
 #include <daos/pool.h>
+#include <daos/container.h>
 
 static char *progname;
 
 void print_usage(void)
 {
-	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-x] "
+	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-xwi] "
 			"[-h handles_per_pool]\n", progname);
 }
 
+int pause_for_keypress()
```

Comment:
(style)  Bad function definition - int pause_for_keypress() should probably be int pause_for_keypress(void)